content/reference/rest-api/engine-diagrams/create.md

```diff
@@ -0,0 +1,71 @@
+---
+title: 'Engine Diagrams: create'
+weight: 10
+
+menu:
+  main:
+    name: 'create'
+    identifier: 'rest-api-engine-diagrams-create'
+    parent: 'rest-api-engine-diagrams'
+    pre: 'Creates or updates an engine diagram.'
+---
+
+Creates a new engine diagram if it doesn't exist yet or updates an engine diagram with a new version.
+
+# Request
+
+## HTTP Request
+
+```
+PUT https://cawemo.com/api/v1/engine-projects/:project-name/engine-diagrams
+```
+
+## Path Parameters
+
+<table class="table table-striped">
+  <tr>
+    <th>Name</th>
+    <th>Type</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>project-name</td>
+    <td>string</td>
+    <td>Name of the engine project the diagram is located in.</td>
```

Comment:
May sound a bit better, but feel free to dismiss the suggestion if you don't like it.
```suggestion
    <td>Name of the engine project that contains the diagram.</td>
```

Reply:
And if this works, there is at least one more place that uses the same phrasing.